lib/kennel/models/monitor.rb

```diff
@@ -27,7 +27,7 @@ class Monitor < Base
         multi: ->  { type != "query alert" || query.include?(" by ") },
         critical_recovery: -> { nil },
         warning_recovery: -> { nil },
-        require_full_window: -> { true }
+        require_full_window: -> { require_full_window_default(type, query) }
```

Comment:
can just define `require_full_window` and use type + query inside

Reply:
Downside to this is that it's not that nice to test, but I can refactor if you'd like.

Reply:
yeah it's a bit harder to test, but I'd prefer simple code over simple tests :)

Reply:
basically needs a new monitor for each test-case

Reply:
`Monitor.new(query: -> { "max" })` etc